spec/default_facts.yml

```diff
@@ -2,7 +2,13 @@
 #
 # Facts specified here will override the values provided by rspec-puppet-facts.
 ---
-concat_basedir: "/tmp"
 ipaddress: "172.16.254.254"
+ipaddress6: "FE80:0000:0000:0000:AAAA:AAAA:AAAA"
```

Comment:
reasonable improvement but also seems entirely unrelated to adding Sensitive type